src/trnpy/trnsys/simulation.py

```diff
@@ -150,6 +150,21 @@ def get_current_time(self) -> float:
 
         return value
 
+    def get_start_time(self) -> float:
+        """Return the start time of the simulation.
+
+        Returns:
+            float: The simulation start time.
+
+        Raises:
+            SimulationNotInitializedError
+        """
+        (value, error_code) = self.lib.get_start_time()
+        if error_code:
+            raise SimulationNotInitializedError
```

Comment:
It'd be nice to include the error code in this exception.

We may have had this conversation a few months ago, but will it always error because the simulation isn't initialized? If so, it there a way to prevent a client from even getting a `Simulation` object it that happens? That way we can avoid throwing this error entirely and make these `get_*` methods `@property`s.

Reply:
One of the consequences of #61 is that it's no longer possible to get a `Simulation` without a successfully initialized simulation, because we moved deck loading into the creation of a `LoadedTrnsysLib`.  The TRNSYS kernel still checks that the deck was initialized, but we can safely ignore the returned error code because we know it will always be 0.

Considering that, I put together #84.  Take a look a let me know what you think!